editor/project_manager/project_dialog.cpp

```diff
@@ -352,7 +352,7 @@ void ProjectDialog::_install_path_changed() {
 
 void ProjectDialog::_browse_project_path() {
 	String path = project_path->get_text();
-	if (path.is_empty()) {
+	if (path.is_empty() || path.is_relative_path()) {
```

Comment:
If the `path` is relative, should it be appended to the `default_project_path`?

Reply:
I don't know. I think relative path isn't accounted for at all
https://github.com/godotengine/godot/blob/5675c76461e197d3929a1142cfb84ab1a76ac9dd/editor/project_manager/project_dialog.cpp#L165

Reply:
btw empty path is also considered relative, so you could remove the empty check.

Reply:
Do i change line-165 as well?

Reply:
You could.

Reply:
Updated